src/gt4sd/training_pipelines/regression_transformer/core.py

```diff
@@ -0,0 +1,152 @@
+#
+# MIT License
+#
+# Copyright (c) 2022 GT4SD team
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+"""Regression Transformer training utilities."""
+import os
+from dataclasses import dataclass, field
+from typing import Optional
+
+from ..core import TrainingPipelineArguments
+from .utils import TransformersTrainingArgumentsCLI
+
+DATA_ROOT = os.path.join(
```

Comment:
cache is dynamic see here: https://github.com/GT4SD/gt4sd-core/blob/09a7b1d34233f737fa763d0b23a3fab1cc9b4e29/src/gt4sd/configuration.py#L46.
We should get the path from the settings.